sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubHelpers.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.beam.sdk.io.gcp.pubsub;
+
+import java.io.IOException;
+
+/** A set of helper functions and classes used by {@link PubsubIO}. */
+abstract class PubsubHelpers {
+
+  private static final String RESOURCE_NOT_FOUND_ERROR = "Pubsub %1$s %2$s not found for project"
+      + " \"%3$s\". Please create the %1$s %2$s before pipeline"
+      + " execution. If the %1$s is created by an earlier stage of the pipeline, this"
+      + " validation can be disabled using #withoutValidation.";
+
+  static void validateTopicExists(PubsubClient pubsubClient, String projectId, String topicName)
```

Comment:
I don't think this is a hugely useful wrapper, given that we can just call the underlying method and have a precise (based on call location) message